scripts/local/constants.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+# Use lower_case variables in the scripts and UPPER_CASE variables for override
+# Use the constants.sh for env overrides
+
+TELEPORTER_PATH=$(
+    cd "$(dirname "${BASH_SOURCE[0]}")"
+    cd ../.. && pwd
+)
+
+# Set the PATHS
+GOPATH="$(go env GOPATH)"
+
+# Current branch
+current_branch=$(git symbolic-ref -q --short HEAD || git describe --tags --exact-match || true)
```

Comment:
what is this used for? Does it need to be capitalized for use elsewhere

Reply:
Not used currently. This was taken from `constants.sh` in `awm-relayer`, which took it from `avalanchego`